storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3823,18 +3868,22 @@ class Rdb_transaction {
 
   void incr_update_count(TABLE_TYPE table_type) {
     assert(!is_ac_nl_ro_rc_transaction());
+    assert(table_type != TABLE_TYPE::USER_TABLE || get_snapshot_ts() != 0);
```

Comment:
maybe add similar check into incr_insert_count()?

Reply:
That one might be called without snapshot during bulkloading. I
have added a comment there.